src/App.tsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+
+import { Pagination } from './components/Pagination';
+
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
+const units = getNumbers(1, 42)
```

Comment:
Consider renaming 'units' to 'itemsList' or another descriptive name to clarify its purpose 🤔